### PR TITLE
Update install-of-alternatives.md to remove Starlight and some other changes

### PR DESCRIPTION
**Pull request by Haskellered**
- Remove Starlight
- Replace FO and SO with Additive and Adrenaline, respectively.
- Add note to Rubidium section mentioning Embeddium
- Fix Forge heading size

### CODE REVIEW

File: src/wiki/getting-started/install-of-alternatives.md [hunk 1]
```diff
@@ -33,21 +33,21 @@ To install in Prism Launcher, create an instance with the **Fabric (or Quilt)**
...
 
-[See the installation instructions](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#prism-launcher) for Prism Launcher.
+[See the installation instructions](https://additive.intergrav.xyz/downloads/#installing-with-third-party-launchers) for Prism Launcher.
 
-### <img src="https://cdn-raw.modrinth.com/data/BYfVnHa7/366754519a9702b43f0c9462b24de2757393efb9.png" height="20"> Simply Optimized
+### <img src="https://cdn.modrinth.com/data/BYN9yKrV/icon.png" height="20"> Adrenaline
 
-[Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed with just optimization in mind. SO has better out-of-the-box performance than Fabulously Optimized, but it doesn't come with the QoL mods or full OptiFine parity you would see in Fabulously Optimized, so you're expected to add any additional mods you want yourself.
+[Adrenaline](https://modrinth.com/modpack/adrenaline) is a modpack designed with just optimization in mind. Adrenaline has better out-of-the-box performance than Additive, but it doesn't come with the QoL mods or full OptiFine parity you would see in Additive, so you're expected to add any additional mods you want yourself.
```
Comment (Madis0) [inline src/wiki/getting-started/install-of-alternatives.md:50]: Why the change?

**Review by Madis0 — CHANGES_REQUESTED** [review]

**Comment by Imzxhir** [comment]
As much as I respect your decision on removing FO and SO, I don’t think they should be replaced with Adrenaline and Additive.

Don’t get me wrong they are both great modpacks by Devin, but at the same time it seems we are explicitly promoting his packs and saying that his packs provide the best performance. However, each user would have different results on performance, some would get high and some would get worse compared to other packs.

**Review by Kichura — CHANGES_REQUESTED** [review]
Please do not switch fabulously optimized with additive and adrenaline respectively, there is a clear reason for that.

*(EDIT: We don't want this wiki to make use of alternative packs that look simple on paper but in practice make performance start to vary for other people which zxhir's comment is morely sensable of)*

**Review by osfanbuff63 — COMMENTED** [review]
Personally, I believe that *removing* FO and SO is a bad idea and would cause much controversy. I also agree that Devin's packs are great packs, but it's not fair to completely replace it after all this time.

To come to a fair compromise, maybe we could have both packs listed in both spots.

*Disclaimer: I am a community moderator for FO, but this comment should be free of that bias, as I also like Devin's modpacks for certain uses.*

**Comment by Haskellered** [comment]
Quickly quoting ZekeZ from the discord:
image
Should I just remove it? Or undo it?

**Comment by Kichura** [comment]
> Quickly quoting ZekeZ from the discord:
> image
> Should I just remove it? Or undo it?

Suggest undoing it.

**Comment by TayouVR** [comment]
I don't really see the point of these changes. Adding Devins packs and Embeddium on top of whats already there, sure!
But removing the existing packs (and mods, including starlight)... I really can't see the benefit. 
Sure; some mods may have incompatibilities with others, but they can still be useful for people with limited hardware capabilities.

If you just added what you did without removing any existing things, I feel like that would be better. Only time where I would remove something would be if A: its discontinued, B: its unstable (which none of the things are, there are only few incompatibilities), C: there are issues with the license or author, or D: it doesn't actually do what it says on the box (e.g. doesn't help with performance, or OF compat)

**Comment by Haskellered** [comment]
> I don't really see the point of these changes. Adding Devins packs and Embeddium on top of whats already there, sure! But removing the existing packs (and mods, including starlight)... I really can't see the benefit. Sure; some mods may have incompatibilities with others, but they can still be useful for people with limited hardware capabilities.

I'll re-add starlight (as well as the rest), but note that it's only useful for older versions, then.

**Comment by intergrav** [comment]
Please don't replace SO and FO with my modpacks. I'd much prefer if they were added as alternatives

**Comment by IThundxr** [comment]
> I don't really see the point of these changes. Adding Devins packs and Embeddium on top of whats already there, sure! But removing the existing packs (and mods, including starlight)... I really can't see the benefit. Sure; some mods may have incompatibilities with others, but they can still be useful for people with limited hardware capabilities.
> 
> If you just added what you did without removing any existing things, I feel like that would be better. Only time where I would remove something would be if A: its discontinued, B: its unstable (which none of the things are, there are only few incompatibilities), C: there are issues with the license or author, or D: it doesn't actually do what it says on the box (e.g. doesn't help with performance, or OF compat)

Startlight is pretty unstable in 1.20, it crashes with almost any mod that does some funny rendering stuff which alot do so it might just be better to remove it since it is infact as said on its mod page, irrelevent now

**Pull request closed without merging**